addons/point_of_sale/tools/posbox/configuration/requirements.txt

```diff
@@ -0,0 +1,72 @@
+cryptography==42.0.8 ; python_version >= '3.12'  # (Noble) min 41.0.7, pinning 42.0.8 for security fixes
+dbus-python==1.3.2
+decorator==4.4.2  ; python_version < '3.11'  # (Jammy)
+decorator==5.1.1  ; python_version >= '3.11'
+docutils==0.17 ; python_version < '3.11'  # (Jammy)
+docutils==0.20.1 ; python_version >= '3.11'
+evdev==1.6.0
+freezegun==1.2.1
+gatt
+geoip2==2.9.0
+ghostscript==0.7 ; sys_platform == 'win32'
+Jinja2==3.0.3 ; python_version <= '3.10'
+Jinja2==3.1.2 ; python_version > '3.10'
+libsass==0.20.1 ; python_version < '3.11'
+libsass==0.22.0 ; python_version >= '3.11'  # (Noble) Mostly to have a wheel package
+lxml==4.8.0 ; python_version <= '3.10'
+lxml==4.9.3 ; python_version > '3.10' and python_version < '3.12' # min 4.9.2, pinning 4.9.3 because of missing wheels for darwin in 4.9.3
+lxml==5.2.1; python_version >= '3.12' # (Noble - removed html clean)
+Mako==1.1.4
+mock==4.0.3
+netifaces==0.11.0
+num2words==0.5.13
+passlib==1.7.4 # min version = 1.7.2 (Focal with security backports)
+Pillow==9.0.1 ; python_version <= '3.10'
+Pillow==9.4.0 ; python_version > '3.10' and python_version < '3.12'
+polib==1.1.1
+psutil==5.9.0 ; python_version <= '3.10'
+psutil==5.9.4 ; python_version > '3.10' and python_version < '3.12'
+psutil==5.9.8 ; python_version >= '3.12' # (Noble) Mostly to have a wheel package
+psycopg2==2.9.2 ; python_version == '3.10' # (Jammy)
+psycopg2==2.9.5 ; python_version == '3.11'
+psycopg2==2.9.9 ; python_version >= '3.12' # (Noble) Mostly to have a wheel package
+pycups==2.0.1
+pydot==1.4.2
+PyKCS11==1.5.16 ; sys_platform == 'win32'
+PyKCS11 ; sys_platform != 'win32'
+pyopenssl==21.0.0 ; python_version < '3.12'
+pyopenssl==24.1.0 ; python_version >= '3.12' # (Noble) min 23.2.0, pinned for compatibility with cryptography==42.0.8 and security patches
+PyPDF2==1.26.0 ; python_version <= '3.10'
+PyPDF2==2.12.1 ; python_version > '3.10'
+pyserial==3.5
+pysmb==1.2.9.1
+python-dateutil==2.8.1 ; python_version < '3.11'
+python-dateutil==2.8.2 ; python_version >= '3.11'
+python-ldap==3.4.0 ; sys_platform != 'win32' and python_version < '3.12' # min version = 3.2.0 (Focal with security backports)
+python-ldap==3.4.4 ; sys_platform != 'win32' and python_version >= '3.12'  # (Noble) Mostly to have a wheel package
+python-stdnum==1.17 ; python_version < '3.11'  # (jammy)
+python-stdnum==1.19 ; python_version >= '3.11'
+pytz  # no version pinning to avoid OS perturbations
+pyusb==1.2.1
+qrcode==7.3.1 ; python_version < '3.11'  # (jammy)
+qrcode==7.4.2 ; python_version >= '3.11'
+reportlab==3.6.8 ; python_version <= '3.10'
+reportlab==3.6.12 ; python_version > '3.10' and python_version < '3.12'
+reportlab==4.1.0 ; python_version >= '3.12' # (Noble) Mostly to have a wheel package
+requests==2.25.1 ;  python_version < '3.11' # versions < 2.25 aren't compatible w/ urllib3 1.26. Bullseye = 2.25.1. min version = 2.22.0 (Focal)
+requests==2.31.0 ; python_version >= '3.11' # (Noble)
+rjsmin==1.1.0
+RPi.GPIO
```

Comment:
this lib probably make no sense to pip install on windows IoT (with other on this list, but this one is quite obvious)